### PR TITLE
Fix #1970

### LINKED ISSUE #1970: Tutorial Fetch Data Dynamically - Unable to load data in sandbox. Request is cancelled.
I went through the tutorial and found two issues with loading data.

https://dart.dev/tutorials/web/fetch-data#using-getString-function
https://dart.dev/tutorials/web/fetch-data#making-a-get-request

In both cases, the request fails and the data is not loaded. In network monitoring, I see both of those requests as canceled.

image

### PR DESCRIPTION
**Pull request by dhanushuUzumaki**
Updated to use new Gists (URLs changed to dart.dev)

### CODE REVIEW

File: src/_tutorials/web/fetch-data.md [hunk 1]
```diff
@@ -414,7 +414,7 @@ and loads the file.
 
 **Try it!** Click **Run** and then click the **Get portmanteaux** button.
 
-{% comment %} https://gist.github.com/chalin/6b76bce8d46986e624f4e82925c48287 {% endcomment %}
+{% comment %} https://gist.github.com/kwalrath/70bb4db7bea44663206e7a9c111f31d2 {% endcomment %}
 <iframe
 src="{{site.dartpad-embed-html}}?id=6b76bce8d46986e624f4e82925c48287"
```
Comment (kwalrath) [inline src/_tutorials/web/fetch-data.md]: You need to update 6b76bce8d46986e624f4e82925c48287 in this line to be 70bb4db7bea44663206e7a9c111f31d2 instead.

File: src/_tutorials/web/fetch-data.md [hunk 2]
```diff
@@ -470,7 +470,7 @@ and use the [send()][] method to make the request.
 This section rewrites the portmanteaux code to explicitly construct
 an HttpRequest object.
 
-{% comment %} https://gist.github.com/chalin/c387e454cb751ab0632c68ccbbf94d12 {% endcomment %}
+{% comment %} https://gist.github.com/kwalrath/f8aa7310e96d359c5f68343c9ccba8ed {% endcomment %}
 <iframe
 src="{{site.dartpad-embed-html}}?id=c387e454cb751ab0632c68ccbbf94d12"
```
Comment (kwalrath) [inline src/_tutorials/web/fetch-data.md]: c387e454cb751ab0632c68ccbbf94d12 -> f8aa7310e96d359c5f68343c9ccba8ed

File: src/_tutorials/web/fetch-data.md (hunk 1 above)
Comment (dhanushuUzumaki) [reply src/_tutorials/web/fetch-data.md]: Oops Sorry I just saw the comment tag around that. Will update.

**Comment by kwalrath** [comment]
For the record, the gists added in this PR were created by @Arun8295 (#1800), who should get credit for contributing to this PR.

**Pull request merged**